perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -480,6 +484,34 @@ public static boolean isAuthorizedForAttribute(PerunSession sess, ActionType act
 				if(isAuthorized(sess, Role.FACILITYADMIN, f)) return true;
 			}
 			if(roles.contains(Role.SELF)); //Not allowed
+		} else if(ues != null) {
+			User sessUser = sess.getPerunPrincipal().getUser();
+			User uesUser;
+			try {
+				uesUser = getPerunBlImpl().getUsersManagerBl().getUserById(sess, ues.getUserId());
+			} catch (UserNotExistsException ex) {
+				return false;
+			}
+			if(ues.getUserId() == sessUser.getId() ) return true;
+			if(roles.contains(Role.FACILITYADMIN)) {
+				List<Facility> facilities = getPerunBlImpl().getFacilitiesManagerBl().getAssignedFacilities(sess, uesUser);
+				for(Facility f: facilities) {
+					if(isAuthorized(sess, Role.FACILITYADMIN, f)) return true;
+				}
+			}
+			if(roles.contains(Role.VOADMIN) || roles.contains(Role.VOOBSERVER)) {
+				List<Vo> vos = getPerunBlImpl().getUsersManagerBl().getVosWhereUserIsMember(sess, uesUser);
+				for(Vo v: vos) {
+					if(isAuthorized(sess, Role.VOADMIN, v)) return true;
+					if(isAuthorized(sess, Role.VOOBSERVER, v)) return true;
+				}
+			}
+			if(roles.contains(Role.GROUPADMIN)) {
+				List<Group> groups = getPerunBlImpl().getUsersManagerBl().getGroupsWhereUserIsAdmin(sess, sessUser);
```

Comment:
I agree. Reason for this is that group manager can add any VO member to his group anyway.